mkdocs/utils.py

```diff
@@ -226,3 +228,29 @@ def path_to_url(path):
         return path
 
     return pathname2url(path)
+
+
+def convert_markdown(markdown_source, extensions=[], extension_configs={}):
```

Comment:
use extensions=None and extensions_configs=None.

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments